script/diaspora-dev

```diff
@@ -235,8 +235,15 @@ dia_get_db() {
 
 dia_build() {
   if [ $# -gt 0 ] && [ "$1" == "--no-cache" ]; then nocache="--no-cache"; fi
+
   # Build the diaspora Docker container (diaspora:dev-latest)
-  dia_docker_compose build --pull $nocache diaspora
+  if type podman &> /dev/null && [[ "${DOCKER_HOST}" == *"podman"* ]]; then
```

Comment:
I think `command -v` might be more common than `type`, but no objections here, just a note.

Reply:
Sure. I don't really mind, I just picked one that worked for me at another place, but I switched it to `command -v`. :+1: